uint8   u_switch_index
bool    is_switch_high
---
